doc/sphinx/Makefile

```diff
@@ -9,8 +9,19 @@ SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
 VENVDIR       = $(SPHINXDIR)/venv
-PA11Y         = $(SPHINXDIR)/node_modules/pa11y/bin/pa11y.js --config $(SPHINXDIR)/pa11y.json
+PA11Y         = $(SPHINXDIR)/node_modules/pa11y-ci/bin/pa11y-ci.js --config $(SPHINXDIR)/pa11y.json
```

Comment:
Does `-ci` give us significant boost now? Should we propose to the starter pack first?

Reply:
I haven't yet compared performance, but the readability improvements are worth it alone

Reply:
Sure, I just mean diverging from the starter pack is going to make it harder to refresh it, so if we think this an improvement, we should propose it upstream sooner than later.

Reply:
Yeah, I'll check real quick if there's a reason the docs team didn't use it (license or similar), if not, I'll open a PR